addons/account_audit_trail/models/mail_message.py

```diff
@@ -0,0 +1,78 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from markupsafe import Markup
+
+from odoo import api, fields, models, _
+from odoo.exceptions import UserError
+
+
+class Message(models.Model):
+    _inherit = 'mail.message'
+
+    account_audit_log_preview = fields.Html(string="Description", compute="_compute_account_audit_log_preview")
+    account_audit_log_move_id = fields.Many2one(
+        comodel_name='account.move',
+        string="Journal Entry",
+        compute="_compute_account_audit_log_move_id",
+        search="_search_account_audit_log_move_id",
+    )
+    show_audit_log = fields.Boolean(compute="_compute_account_audit_log_move_id", search="_search_show_audit_log")
+
+    @api.depends('body', 'subject', 'tracking_value_ids', 'subtype_id')
+    def _compute_account_audit_log_preview(self):
+        for message in self:
+            title = message.subject or message.preview
+            tracking_value_ids = message.sudo().tracking_value_ids
```

Comment:
Why is `sudo` alright here?
Aren't we giving access to stuff one shouldn't have access to?

Reply:
hello @william-andre ,

Right now only system admin can access the `tracking_value_ids`.
I can remove sudo from here and define access right for `group_account_manager` to read the `tracking_value_ids`.
so `group_account_manager` can access the audit trail.
what is your opinion on this ?

Reply:
You should do something like this instead https://github.com/odoo/odoo/blob/db61d9e025909bc79dfdb565e8d29d0e3532ec59/addons/mail/models/mail_message.py#L922